app/controllers/art_labels_controller.rb

```diff
@@ -8,6 +8,14 @@ def index
 
   def show
     @art_label = ArtLabel.find(params[:id])
+
+    if @art_label.user.nil?
+      @creator = "User no longer exists"
```

Comment:
Coveralls is yelling at you about how this line is never tested, fix this!

Reply:
Al and Bobby are fixing it in their pull

Reply:
Are they fixing that other coveralls issue?

Reply:
The untested line in the application controller for raising the "Not an Admin" error?